mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -183,6 +228,10 @@ class DistanceSensorPlugin : public MavRosPlugin {
 void DistanceSensorItem::range_cb(const sensor_msgs::Range::ConstPtr &msg)
 {
 	uint8_t type = 0;
+	uint8_t covariance_ = 0;
+
+	if (cov_is_def == true) covariance_ = covariance;
+	else covariance_ = uint8_t(calculate_variance(msg->range * 1E2));	// in cm
```

Comment:
@vooon I'm doing some testing to see if the value of variance is acceptable but seems like the routine is not even entering in `calculate_covariance()`. I'm using `ROS_INFO_STREAM` to do the debug, and `ROS_INFO_STREAM("Variance:", covariance_)` prints `Variance: ` (where if I put a static value like `0.0`, it prints  `Variance: 0.0`).
Putting a `ROS_INFO_STREAM` statement inside `calculate_covariance(float range)` method does print a thing. What can be wrong?